miden-lib/asm/sat/note_setup.masm

```diff
@@ -0,0 +1,38 @@
+use.miden::sat::prologue
+
+# MEMORY OFFSETS
+# =================================================================================================
+
+# BOOK KEEPING
+# -------------------------------------------------------------------------------------------------
+const.CURRENT_CONSUMED_NOTE_PTR=3
+const.CONSUMED_NOTE_INPUTS_HASH_OFFSET=3
+
+# NOTE SETUP SCRIPT
+# =================================================================================================
+
+export.note_setup_script.5
```

Comment:
nit: I wonder if a verb-based name would be better. Maybe something like `perpare_note`?

Reply:
Good call. Any suggestions on how we should name the prologue procedure?  It's currently named `transaction_prologue`.

Reply:
We could say something like `prepare_transaction` but not sure if it is a big improvement.